src/components/Map/constant.tsx

```diff
@@ -0,0 +1 @@
+export const MAP_KEY = "AIzaSyCdiWIZIVkqOw7pmXBEMria1Qe4TxgKbo0";
```

Comment:
Should we each keep the key in our .env file instead? (https://stackoverflow.com/questions/48699820/how-do-i-hide-api-key-in-create-react-app)
But apparantly, .env is not secure either LOL. So if we're going to set up the backend to make those API calls, then we might be fine with storing as constant for now....

Reply:
I got an email yesterday from Google when they found the key in this public repo lol. 
Yea, .env will make it slightly harder to get the key. Let's move the api key to somewhere else when we are setting up the backend.